docs/content/integrations/dbt.mdx

```diff
@@ -1,379 +1,209 @@
+```python
+from dagster import with_resources
 from dagster_dbt import dbt_cli_resource
 
-from dagster import job
-
-config = {"dbt_executable": "path/to/dbt/executable"}
-
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
+dbt_assets = with_resources(
+    load_assets_from_dbt_project(...),
+    {
+        "dbt": dbt_cli_resource.configured(
+            {"project_dir": "path/to/dbt_project"}
+        ),
+        "pandas_df_manager": pandas_io_manager.configured(
+            {"con_string": "..."}
+        )
+    }
+)
 ```
 
-**dbt CLI: Select specific models to run**
+## Understanding AssetKeys
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_select_models endbefore=end_marker_dbt_cli_config_select_models dedent=4
-from dagster_dbt import dbt_cli_resource
+In Dagster, each asset has an asset key to identify it. Dagster automatically generates these keys for each dbt node in the project as well as the sources for each node.
 
-from dagster import job
+For models, seeds, and snapshots, the default asset key will be the configured schema for that node (if any), concatenated with the name of the node.
 
-config = {"models": ["my_dbt_model+", "path.to.models", "tag:nightly"]}
+For example, if you have configured a [custom schema](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas) for a subdirectory in your `dbt_project.yml` file:
 
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
+```yaml
+models:
+  my_project:
+    marketing:
+      +schema: marketing
 ```
 
-For more details, [visit the official documentation on dbt's node selection syntax](https://docs.getdbt.com/reference/node-selection/syntax).
-
-**dbt CLI: Exclude specific models**
+Then the asset key for a model named "some_model" will be `marketing/some_model`. If you have not configured a custom schema, then the asset key will simply be `some_model`.
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_exclude_models endbefore=end_marker_dbt_cli_config_exclude_models dedent=4
-from dagster_dbt import dbt_cli_resource
-
-from dagster import job
+For sources, the default asset key will be the name of the source concatenated with the name of the source table.
 
-config = {"exclude": ["my_dbt_model+", "path.to.models", "tag:nightly"]}
+For example, the source table defined in the following `sources.yaml` will be `jaffle_shop/orders`:
 
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
+```yaml
+sources:
+  - name: jaffle_shop
+    tables:
+      - name: orders
 ```
 
-For more details, [visit the official documentation on dbt's node selection syntax](https://docs.getdbt.com/reference/node-selection/syntax).
-
-**dbt CLI: Set key-values for dbt vars**
+### Adding a prefix to AssetKeys
```

Comment:
```suggestion
### Adding a prefix to asset keys
```

Reply:
Just saw your note about this section feeling misplaced - I agree.

What if we moved this up to come after the **dbt models and software-defined assets** section? This section relates to assets themselves, so it feels like that might be a good place for it instead of at the end. Especially with the title `Understanding asset keys`.

Thoughts?